sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -895,10 +895,16 @@ Table 10. Queue info query
 
 |===
 
-A default constructed event is returned when a user submits a command-group to
-a queue in the recording state. These events have status
-`info::event_command_status::complete` and a user waiting on them will return
-immediately.
+Events returned from queue submissions when a queue is in the recording state
```

Comment:
Although we're not saying a default constructed state, should we still specify what [info::event_command_status](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.event.info) state the event will be in. Anythig other than complete I think wouldn't make sense.

E.g

"Events returned from queue submissions when a queue is in the recording state **have a complete execution status** and may only be user as parameters..."